lib/tasks/evm_application.rb

```diff
@@ -102,11 +102,12 @@ def self.output_workers_status(servers)
            w.miq_server_id,
            w.queue_name || w.uri,
            w.started_on && w.started_on.iso8601,
-           w.last_heartbeat && w.last_heartbeat.iso8601]
+           w.last_heartbeat && w.last_heartbeat.iso8601,
+           ActionView::Base.new.number_to_human_size(w.proportional_set_size || w.memory_usage) || ""]
```

Comment:
Prefer `.to_s(:human_size)` over calling out to ActionView directly. http://edgeguides.rubyonrails.org/active_support_core_extensions.html#formatting

```ruby
(w.proportional_set_size || w.memory_usage || "").to_s(:human_size)
```